src/material-experimental/mdc-chips/chip.ts

```diff
@@ -257,6 +257,12 @@ export class MatChip extends _MatChipMixinBase implements AfterContentInit, Afte
     },
     hasLeadingIcon: () => { return !!this.leadingIcon; },
     setAttr: (name: string, value: string) => {
+      // MDC is currently using this method to set aria-checked on choice and filter chips,
+      // which in the MDC templates have role="checkbox" and role="radio" respectively.
+      // We have role="option" on those chips instead, so we do not want aria-checked.
+      if (name === 'aria-checked') {
```

Comment:
Could you file an issue on the MDC repo about this? I would consider it overzealous.

Reply:
Done. https://github.com/material-components/material-components-web/issues/4910